src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -312,14 +356,20 @@ else if (s.team.equals(team.name)) {
             if (isLastStudent) {
                 sections.add(section);
                 if (!section.name.equals(Const.DEFAULT_SECTION)) {
-                    cdd.stats.sectionsTotal++;
+                    courseDetails.stats.sectionsTotal++;
                 }
             }
         }
         
         return sections;
     }
     
+    /**
+     * Gets a list of {@link SectionDetailsBundle sectionDetails} for a given course using courseId.
```

Comment:
Will need you to also edit this to be `section details` and in other descriptions too